po/de.po

```diff
@@ -1161,12 +1200,15 @@ msgid ""
 "additional keywords you can use; refer to the user manual to learn more "
 "about its features and syntax."
 msgstr ""
+"Der Schnelleintrag ist der schnellste Weg um neue Aufgaben ohne "
+"Unterbrechungen direkt im Hauptfenster zu erstellen. Es besitzt spezielle "
```

Comment:
"Er" instead of "Es" for grammatical correctness because "Schnelleintrag" is masculine.